server.js

```diff
@@ -40,14 +43,74 @@ app.use('/api', router);
 
 router.get('/feedItems', function (req, res) {
     console.log('GET Feed Items');
-    // get all the feedItems
-    FeedItem.find({}, function(err, feedItems) {
-        if (err) throw err;
-
-        // array of feed items
-        console.log(feedItems);
-        res.json(feedItems);
-    });
+    var requestedCount = parseInt(req.query.count);
+    var requestedIndex = parseInt(req.query.index);
```

Comment:
If we have parseInt problems, do we get NaN and therefore `if(NaN)` evaluates to `false`?